modules/node_groups/locals.tf

```diff
@@ -24,6 +24,7 @@ locals {
       pre_userdata                  = var.workers_group_defaults["pre_userdata"]
       additional_security_group_ids = var.workers_group_defaults["additional_security_group_ids"]
       taints                        = []
+      timeouts                      = var.workers_group_defaults["timeouts"]
```

Comment:
`var.workers_group_defaults["timeouts"]` is not defined anywhere so this is always null